addons/web/static/src/core/assets.js

```diff
@@ -30,13 +31,23 @@ class AssetsLoadingError extends Error {}
  * @typedef {Object} LoadedBundle
  * @property {XMLDocument} templates an XML document containing the owl
  *      templates defined in that bundle
+ * @property {Promise<string> | string} js a promise or the resolution of that promise that signifies that the js has been loaded
+ * @property {Promise<string> | string} css a promise or the resolution of that promise that signifies that the css has been loaded
+ * @property {Promise<>}
```

Comment:
Echoing my comment on loadJS and loadCSS returning strings: I don't think these keys need to be present at all since their values cannot be used sanely, and the caller knows which parts of the bundle they requested to be loaded so there is nothing useful that could be inferred from the presence of the keys themselves if we remove the strings. In the end I would like to also remove the XML from being returned, and load the templates into the QWeb engine transparently, but since we currently have multiple QWeb instances, I thought it was better to let the caller handle that part for now.